exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -127,18 +141,33 @@ class OTLPExporterMixin(
 
     def __init__(
         self,
-        endpoint: str = "localhost:55680",
+        endpoint: Optional[str] = None,
+        insecure: Optional[bool] = None,
         credentials: ChannelCredentials = None,
         metadata: Optional[Tuple[Any]] = None,
     ):
         super().__init__()
 
+        endpoint = (
+            endpoint
+            or Configuration().EXPORTER_OTLP_ENDPOINT
+            or "localhost:55680"
+        )
+
+        if insecure is None:
+            insecure = Configuration().EXPORTER_OTLP_INSECURE
+        if insecure is None:
+            insecure = True
```

Comment:
Isn't the default value `False`?

Reply:
In current spec it is `False`. I [opened a proposal PR](https://github.com/open-telemetry/opentelemetry-specification/pull/975) for changing default to be `True` which feels more natural to have, otherwise supplying certificate files becomes mandatory.

Reply:
I would prefer the implementation to follow the specs AS OF NOW. If the PR in specs ever gets merged, then we can create a followup PR to change that. That way we avoid confusion (like how I was confused just now).